CesiumGeospatial/src/BoundingRegionBuilder.cpp

```diff
@@ -123,4 +125,31 @@ bool BoundingRegionBuilder::expandToIncludePosition(
   return modified;
 }
 
+bool BoundingRegionBuilder::expandToIncludeGlobeRectangle(
+    const GlobeRectangle& rectangle) {
+  bool modified = false;
+
+  if (rectangle.getSouth() < this->_rectangle.getSouth()) {
+    this->_rectangle.setSouth(rectangle.getSouth());
+    modified = true;
+  }
+
+  if (rectangle.getNorth() > this->_rectangle.getNorth()) {
+    this->_rectangle.setNorth(rectangle.getNorth());
+    modified = true;
+  }
+
+  if (rectangle.getEast() < this->_rectangle.getEast()) {
+    this->_rectangle.setEast(rectangle.getEast());
+    modified = true;
+  }
```

Comment:
I don't think this will handle the anti-meridian correctly. It'd be good to see some tests for this, particular where either the builder, the rectangle, or both cross the anti-meridian.